cbits/simd-spliced.c

```diff
@@ -50,7 +50,10 @@ int hw_json_simd_main_spliced(
     exit(1);
   }
 
-  uint8_t buffer[W8_BUFFER_SIZE];
+  // align stack buffer on 32 bytes 
+  uint8_t unaligned_buffer[W8_BUFFER_SIZE + 32];
+  uint8_t * buffer = unaligned_buffer + ((uintptr_t)unaligned_buffer & (uintptr_t)0x10);
+  fprintf(stderr, "Buffer is at address %p of size %zu\n", buffer, (size_t) W8_BUFFER_SIZE);
```

Comment:
Perhaps it's better to declare a stack allocated array for alignment and get the compiler to handle it:

```
uint64_t aligned_buffer[W8_BUFFER_SIZE / 8];
uint8_t * buffer = (uint8_t *)aligned_buffer;
```

Unless we need alignment at `32` bytes, in which case we'll have to use your solution.